### PR TITLE
DROOLS-1701 Support for FEEL fn invocation using named parameters, fix numeric constants

### PR DESCRIPTION
**Pull request by evacchi**
~~ this is my first PR please be gentle ~~

jokes aside, I have limited connection, so please do check & `mvn clean verify` before merging!

### CODE REVIEW

File: kie-dmn/kie-dmn-feel/src/main/java/org/kie/dmn/feel/parser/feel11/ParserHelper.java [hunk 1]
```diff
@@ -247,6 +247,17 @@ public static String getOriginalText(ParserRuleContext ctx) {
         return ctx.getStart().getInputStream().getText( interval );
     }
 
+    /**
+     * Sanitizes a number of the form: <code>123e+10</code> and turns it into a valid identifier
+     */
+    public static String toNumericIdentifier(String number) {
```
Comment (tarilabs) [inline kie-dmn/kie-dmn-feel/src/main/java/org/kie/dmn/feel/parser/feel11/ParserHelper.java]: Can you please move this into `CodegenStringUtil` please?

File: kie-dmn/kie-dmn-feel/src/main/java/org/kie/dmn/feel/parser/feel11/ParserHelper.java [hunk 2]
```diff
@@ -247,6 +247,17 @@ public static String getOriginalText(ParserRuleContext ctx) {
...
+        // FIXME quick & dirty sanitization routine
+        return "K_" + number
+                .replace(".", "$point$")
```
Comment (tarilabs) [inline kie-dmn/kie-dmn-feel/src/main/java/org/kie/dmn/feel/parser/feel11/ParserHelper.java]: `replace(".", escapeIdentifier(".")`

Comment (tarilabs) [reply kie-dmn/kie-dmn-feel/src/main/java/org/kie/dmn/feel/parser/feel11/ParserHelper.java]: ^ suggestion please

**Review by tarilabs — COMMENTED** [review]
Thanks a lot!! Just a couple of small details please then I'll merge into the feature branch

File: kie-dmn/kie-dmn-feel/src/main/java/org/kie/dmn/feel/parser/feel11/ParserHelper.java (hunk 2 above)
Comment (evacchi) [reply kie-dmn/kie-dmn-feel/src/main/java/org/kie/dmn/feel/parser/feel11/ParserHelper.java]: so `escapeIdentifier(number)` ?

Comment (tarilabs) [reply kie-dmn/kie-dmn-feel/src/main/java/org/kie/dmn/feel/parser/feel11/ParserHelper.java]: yeah even simpler, dunno why I didn't do that already before.

**Comment by evacchi** [comment]
comments addressed.

**Comment by tarilabs** [comment]
thanks!

**Pull request merged**